ckanext/switzerland/templates/user/list.html

```diff
@@ -4,7 +4,7 @@
   <article class="module">
     <div class="module-content">
       <h1 class="page-heading">
-        {% if h.check_access('user_create') %}
+        {% if h.check_access('sysadmin') %}
```

Comment:
hmmm, this makes me wonder though, why checking the permission does not work. would be good to know if there is something messed up in the permissions or if there is an actual bug. 
Just in case you haven't investigated yet: can you check if the permission-check is actually correct (and fails correctly), but the re-direct is somehow working incorrectly?

Reply:
The auth check for `user_create` just checks whether the config setting `create_user_via_web` is set to true (or `create_user_via_api` if being accessed via the api). It doesn't check whether the user calling it is a sysadmin or has any role/permissions. I think this must be because normally there's no link to create a new user displayed if you're not logged in as a sysadmin (or at all?), so normally the issue does not come up.

Reply:
Ah, sure, totally missed that this is a custom template, in that case it is totally fine. Thank you for taking the time explaining this to me 🤗